README.md

```diff
@@ -1,3 +1,5 @@
 # Test
 this is a test
 test conflict
+
+#add a new commit to master branch to test rebase and merge
```

Comment:
```suggestion
#add a new commit to master branch to test rebase and merge
```
Does it work? Eventually? What have you changed? 
Would share it with detail steps in this note?Thanks!

Reply:
OKay!